roles/setup_repo/tasks/PG_Debian_setuprepos.yml

```diff
@@ -36,13 +38,16 @@
     url: "{{ edb_deb_keys }}"
     state: present
   become: true
+  when: >
+    enable_edb_repo|bool and repo_token|length <
```

Comment:
should be: `enable_edb_repo and repo_token|length < 1`

Reply:
Fixed